src/components/atoms/EmailPrintButton.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+/**
+ * This component is used to create the print and email button
+ */
+export function EmailPrintButton(props) {
```

Comment:
the ActionButton component should meet this requirements with slight modifications. the className prop should be use to override the button styles.

Reply:
Thank you, I removed this component, used ActionButton instead